src/Utilities/timerLayer.hh

```diff
@@ -0,0 +1,16 @@
+#include "caliper/cali.h"
+
+// If TIMER is defined then we want timer functionality
+#ifdef TIMER
+
+#define TIME_FUNCTION CALI_CXX_MARK_FUNCTION
+#define TIME_BEGIN(regionName) CALI_MARK_BEGIN(regionName)
+#define TIME_END(regionName) CALI_MARK_END(regionName)
+
+#else // TIMER
+// Stub TIME macros, when TIME is zero
+#define TIME_FUNCTION
+#define TIME_BEGIN(regionName)
+#define TIME_END(regionName)
+
+#endif // TIMER
```

Comment:
Lets go ahead and have this moved to Utilities/Timer.hh I think it makes sense to keep all of our timer stuff in one place.

Reply:
Since the old timer was completely removed, we could move/rename the new macro-layer to Timer.hh, I'm just not sure if there is a naming convention we could follow for macro classes.

Reply:
Often for MACROS it can be useful to prefix them with the project name for possibly overloaded name, I think in this case the macro names are fine for us.

Lets move this to Timer.hh since this is our only interface now.